metrics_utility/management/commands/build_report.py

```diff
@@ -61,43 +82,34 @@ def init_logging(self):
         handler.setLevel(logging.DEBUG)
         handler.setFormatter(logging.Formatter('%(message)s'))
         self.logger.addHandler(handler)
-        self.logger.propagate = False
-
-    def handle(self, *args, **options):
+        self.logger.propagate = True
+
+    def _parse_param(
+        self,
+        param_name,
+        options,
+    ):
+        param = options.get(param_name)
+        if (options.get('month') is not None or param is None) and param_name != 'ephemeral':
+            return None
+        if param:
+            return parse_date_param(param) if parser.parse(param) else parse_number_of_days(param)
```

Comment:
* `parser.parse` raises an exception, or returns a truthy value, so putting it inside an `if` doesn't work.
* Let's keep `ephemeral` separate from `since` and `until`, the formats are different.
* the logic for ephmeral vs month is too complicated, and no longer used


maybe something like

```suggestion
    def _parse_date_param(self, value, month):
        if month is not None or value is None:
            return None

        return parse_date_param(value)
```